pkgs/applications/graphics/leocad/default.nix

```diff
@@ -14,26 +15,34 @@ set the variable LEOCAD_LIB=/path/to/libs/ or use option -l /path/to/libs/
 
 let
   parts = fetchurl {
-    url = "https://web.archive.org/web/20190715142541/https://www.ldraw.org/library/updates/complete.zip";
+    url = "https://web.archive.org/web/20210705153544/https://www.ldraw.org/library/updates/complete.zip";
     sha256 = "sha256-PW3XCbFwRaNkx4EgCnl2rXH7QgmpNgjTi17kZ5bladA=";
   };
 
 in
 mkDerivation rec {
   pname = "leocad";
-  version = "21.03";
+  version = "21.06";
 
   src = fetchFromGitHub {
     owner = "leozide";
     repo = "leocad";
     rev = "v${version}";
-    sha256 = "sha256-69Ocfk5dBXwcRqAZWEP9Xg41o/tAQo76dIOk9oYhCUE=";
+    sha256 = "1ifbxngkbmg6d8vv08amxbnfvlyjdwzykrjp98lbwvgb0b843ygq";
   };
 
   nativeBuildInputs = [ qmake qttools ];
 
   buildInputs = [ zlib ];
 
+  propagatedBuildInputs = [ povray ];
+
+  patches = [ ./6e7dd2c7-povray.patch ];
+
+  postPatch = ''
+    substituteInPlace qt/lc_renderdialog.cpp --replace "@@NIX_POVRAY_PATH@@" "${povray}/bin/povray"
+  '';
```

Comment:
```suggestion
  patches = (substituteAll {
    src = ./6e7dd2c7-povray.patch;
    inherit povray;
  });
```

Reply:
Please also remove the hash from the patch file